ui/src/app/applications/components/utils.scss

```diff
@@ -31,4 +31,13 @@
 i.utils-health-status-icon {
     min-width: 13px;
     text-align: center;
+}
+
+.status-button {
+    border-radius: 5px;
+    padding: 2px;
+    border: 1px solid $argo-color-gray-5;
+    &:hover {
+        background-color: $argo-color-gray-4;
+    }
 }
```

Comment:
could you add a newline at eof to make github/git tools happy?

Reply:
Done